src/Plugin/ThemeEntityProcessorManager.php

```diff
@@ -55,9 +55,11 @@ public function getEntityData(ContentEntityInterface $entity, $options = []) {
     $processor = $this->createInstance($plugin_id);
 
     $view_mode = isset($options['view_mode']) ? $options['view_mode'] : 'default';
+    $langcode = !empty($options['langcode']) ? $options['langcode'] : NULL;
```

Comment:
We should check if is isset `$options['langcode']`, empty checks if variable is isset but empty. It will throw the exception of undefined key if variable is not set

Reply:
@euphoricMV  switched to isset